distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc

```diff
@@ -21,37 +21,37 @@
 |
 
 |metadata-content-type *[deprecated]*
-|Content type of the resource
+|Content type of the resource.
 |String
 |< 1024 characters
 | 
 
 |metadata-content-type-version *[deprecated]*
-|Version of the metadata content type of the resource
+|Version of the metadata content type of the resource.
 |String
 |< 1024 characters
 | 
 
 |metadata-target-namespace *[deprecated]*
-|Target namespace of the metadata
+|Target namespace of the metadata.
 |String
 |< 1024 characters
 | 
 
 |metadata
-|Additional XML metadata describing the resource
+|Additional XML metadata describing the resource.
 |XML
-|
+|A valid XML string per RFC 4825
```

Comment:
Perhaps add "XML must be well formed but need not necessarily conform to a schema"